code_challenges/code_wars/person.py

```diff
@@ -0,0 +1,6 @@
+class Person:
+    def __init__(self, first_name, last_name, age):
+        self.first_name = first_name
+        self.last_name = last_name
+        self.age = age
+        self.full_name = first_name + " " + last_name
```

Comment:
This is a-okay if you prefer. I almost always prefer f-strings like so `self.full_name = f"{first_name} {last_name}"`